rules/windows/process_creation/proc_creation_win_susp_lolbin_non_c_drive.yml

```diff
@@ -0,0 +1,29 @@
+title: Wscript Execution from Non C Drive
+id: 5b80cf53-3a46-4adc-960b-05ec19348d74
+status: experimental
+description: Detects Wscript or Cscript executing from a drive other than C. This has been observed with Qakbot executing from within a mounted ISO file.
+author: Aaron Herman
+date: 2022/10/01
+references:
+    - https://github.com/pr0xylife/Qakbot/blob/main/Qakbot_BB_30.09.2022.txt
```

Comment:
I can't see an example for what you're trying to detect visiting this link or the link in the PR description. 
Where is an example that doesn't include `C:\` in the command line?

Reply:
Added https://app.any.run/tasks/4985c746-601e-401a-9ccf-ae350ac2e887/